qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -113,12 +116,12 @@ def optimizer(self, optimizer: Optional[Optimizer]) -> None:
         self._optimizer = optimizer
 
     @property
-    def initial_point(self) -> Optional[np.ndarray]:
+    def initial_point(self) -> Optional[Union[np.ndarray, InitialPoint]]:
```

Comment:
In #570 these setters are being deprecated with the underlying algorithm being exposed and the setting of whatever properties it has being direct (otherwise this mirrored the same parameters and needs maintenance etc to keep it in sync with the algorithm that seems burdensome and unnecessary). With this being kept here, to allow these specific points to be used, I guess we will need to document how things work together in that regard.

Reply:
I usually only use setters if I want to do some validation/modification/extraction. As I'm not doing that here, should I just get ahead of the curve and take out this setter before it's merged? I think I only included it to mirror the existing setters anyway.

Reply:
I don't think the getter/setter is necessary for initial_point. So, I've just exposed it directly to match upcoming changes in #570.